com.unity.render-pipelines.core/Runtime/RenderGraph/RenderGraphDefaultResources.cs

```diff
@@ -56,6 +59,7 @@ internal void InitializeForRendering(RenderGraph renderGraph)
             blackUIntTextureXR = renderGraph.ImportTexture(TextureXR.GetBlackUIntTexture());
             blackTexture3DXR = renderGraph.ImportTexture(TextureXR.GetBlackTexture3D());
             whiteTextureXR = renderGraph.ImportTexture(TextureXR.GetWhiteTexture());
+            opaqueBlackTextureXR = renderGraph.ImportTexture(TextureXR.GetOpaqueBlackTexture());
```

Comment:
why "opaqueBlackTextureXR" ? can this be BlackTexture2DXR ?

Reply:
ah I see that there is blackTextureXR, guess I need to read the whole PR to understand why opaqueBlackTextureXR

Reply:
In FinalPass.shader, this is how we combine the after post-process RT with the output:
outColor.xyz = afterPostColor.a * outColor.xyz + afterPostColor.xyz;

This means that the "neutral" output for the pass is [0,0,0,1] and not [0,0,0,0].

I would expect blackTextureXR to be [0,0,0,1], since we also have a "clearTextureXR", but it's not :( that's why I introduced the "opaqueBlackTextureXR".

Reply:
Note that, in other places in Unity black means opaque-black, so I thought about change blackTextureXR to have alpha 1, but I expect this will break many things...

Reply:
> but I expect this will break many things...
ok, got it now. 

currently I don't think changing alpha to 1 for black texture will be harmful.  From what I see, maybe AO or SSGI could be impacted when disabled but I even don't think it have impact. I will suggest let's try it, could you make a separate PR with the change and trigger a yamato just to see what will be the impact? 

Thanks

Reply:
Indeed, changing the alpha to 1 does not look harmful, here is the PR:
https://github.com/Unity-Technologies/Graphics/pull/4253
There are some failed tests on some platforms, but look unrelated (yamato noise).

So I will remove the opaque stuff from here and we should merge the other PR first.